pyartifactory/models/__init__.py

```diff
@@ -1,34 +1,71 @@
+# Copyright (c) 2019 Ananias
+# Copyright (c) 2023 Helio Chissini de Castro
+#
+# Licensed under the MIT license: https://opensource.org/licenses/MIT
+# Permission is granted to use, copy, modify, and redistribute the work.
+# Full license information available in the project LICENSE file.
+#
+# SPDX-License-Identifier: MIT
+

```

Comment:
Can you delete this section ?

Reply:
Well all the added Copyright sections. There is no need to add it on multiple files. The `LICENCE` file is enough

Reply:
removed